### PR TITLE
docs(play): mention feature limitations in Play

### PR DESCRIPTION
**Pull request by huygur**
## Description

closes https://github.com/camunda/play/issues/659

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 1]
```diff
@@ -171,7 +171,10 @@ Prior to the 8.6 release, Play can be accessed by installing the 8.6.0-alpha [He
 
 ### Features
 
-[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- [Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- Currently, Play supports displaying up to 100 flow node instances in the instance history panel, 100 variables in the variables panel, and 100 process instances on the process definition page. To access all related data, you can use Operate.
+- While you can still interact with your process instance in Play—such as completing jobs or publishing messages—you may be unable to resolve incidents if they occur beyond the 100th flow node instance, as Play does not track them. In this case, incident resolution can be managed in Operate.
+- Play doesn't support call activities with feel expressions
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:177]:
```suggestion
- Play doesn't support call activities with FEEL expressions
```
a link here may be helpful if this is the first mention of FEEL in the page- https://docs.camunda.io/docs/components/modeler/feel/what-is-feel/

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 2]
```diff
@@ -171,7 +171,10 @@ Prior to the 8.6 release, Play can be accessed by installing the 8.6.0-alpha [He
 
 ### Features
 
-[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- [Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- Currently, Play supports displaying up to 100 flow node instances in the instance history panel, 100 variables in the variables panel, and 100 process instances on the process definition page. To access all related data, you can use Operate.
+- While you can still interact with your process instance in Play—such as completing jobs or publishing messages—you may be unable to resolve incidents if they occur beyond the 100th flow node instance, as Play does not track them. In this case, incident resolution can be managed in Operate.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:176]:
```suggestion
- While you can still interact with your process instance in Play, such as completing jobs or publishing messages. you may be unable to resolve incidents if they occur beyond the 100th flow node instance, as Play does not track them. In this case, incident resolution can be managed in Operate.
```

**Review by HanselIdes — APPROVED** [review]
LGTM - a few minor suggestions. You do not need to re-request a review

**Review by conceptualshark — DISMISSED** [review]

**Comment by huygur** [comment]
@HanselIdes @conceptualshark I need an approval again to merge, as I pushed a commit to fix a broken link 😅

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 3]
```diff
@@ -171,7 +171,10 @@ Prior to the 8.6 release, Play can be accessed by installing the 8.6.0-alpha [He
 
 ### Features
 
-[Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- [Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable from Play. However, they can be inferred from the output variable, or can be viewed from Operate.
+- Currently, Play supports displaying up to 100 flow node instances in the instance history panel, 100 variables in the variables panel, and 100 process instances on the process definition page. To access all related data, you can use Operate.
+- While you can still interact with your process instance in Play (for example, completing jobs or publishing messages), you may be unable to resolve incidents if they occur beyond the 100th flow node instance, as Play does not track them. In this case, incident resolution can be managed in Operate.
+- Play doesn't support call activities with [FEEL expressions](/components/modeler/feel/what-is-feel.md).
```
Comment (yT0n1) [inline docs/components/modeler/web-modeler/play-your-process.md:177]: Maybe we can generalize this a bit more, but I don't know what the proper wording would be. Or maybe we just list all the items: 
Play does not support Job Types defined through feel expressions, Message correlation keys defined by feel expressions, called process definitions defined by Feel expressions. Maybe I forgot something 😅

**Review by yT0n1 — CHANGES_REQUESTED** [review]
Lgtm, but we should add a general limitation for things defined through feel

**Review by yT0n1 — APPROVED** [review]
Thanks!

**Pull request merged**